src/pudl/transform/ferc1.py

```diff
@@ -2065,27 +2073,35 @@ def multicol_transform_fn_factory(
     col_fn: ColumnTransformFn,
     drop=True,
 ) -> MultiColumnTransformFn:
-    """A factory for creating a table transformation from a column transformation."""
-
-    def tab_fn(df: pd.DataFrame, params: MultiColumnTransformParams) -> pd.DataFrame:
-        drop_col: bool = drop
-        for col_name in params:
-            if col_name in df.columns:
-                new_col = col_fn(col=df[col_name], params=params[col_name])
-                if drop_col:
-                    df = df.drop(columns=col_name)
-                df = pd.concat([df, new_col], axis="columns")
-            else:
-                logger.warning(
-                    f"Expected column {col_name} not found in dataframe during table "
-                    "transform."
-                )
-        return df
-
-    return tab_fn
+    """A factory for creating a multi-column transform function."""
 
-
-def convert_units_col(col: pd.Series, params: UnitConversion) -> pd.Series:
+    class InnerMultiColumnTransformFn(
```

Comment:
why is this called `Inner`?

Reply:
Because it's an "inner" class / function, that's only defined inside of another class / function. This is what it's called when inside one function you define another disposable function, or inside a class you define another class that only exists inside that class.  I needed it to have a different name than the "outer" `MultiColumnTransformFn` so that it didn't clobber that other definition.